app/views/shared/_browse_breadcrumbs.html.erb

```diff
@@ -7,6 +7,7 @@
 
 <div class="browse__breadcrumb-wrapper">
   <div class="govuk-width-container">
+    <div class="browse__breadcrumb-border"></div>
```

Comment:
Shame to have to add another element here just for the border but I can't think of another solution, this seems sensible 👍

Reply:
Yep, this was the existing functionality so I assume it was the best solution for the requirement.